buildSrc/src/main/kotlin/native-targets-conventions.gradle.kts

```diff
@@ -47,6 +48,15 @@ kotlin {
         linuxArm32Hfp()
     }
 
+    // setup tests running in RELEASE mode
+    targets.withType<KotlinNativeTarget>().configureEach {
```

Comment:
From its looks, it replaces all existing targets with the RELEASE build type, so we do not build DEBUG binaries anymore. But I'm not an expert in Kotlin/Native's Gradle.

Reply:
not really. it just adds `RELEASE` test binary, as by default only `DEBUG` test binary is created.
It can be checked by running old existing tasks like `macosArm64Test` - they will depend on `linkDebug*` tasks.

using `binaries.test(listOf(DEBUG, RELEASE))` will cause error trying to create binary which already exists (with `DEBUG`).

Reply:
thnx for explanations